### PR TITLE
Add update speedo instruction

### PR DESCRIPTION
**Pull request by maciejurbanskisaucelabs**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -17,6 +17,12 @@ To download the tool you have to have Node.js installed on your machine. After y
 $ npm install -g speedo
 ```
 
+## Update to newest versionNumber
```
Comment (adambabik) [inline README.md:20]:
```suggestion
## Update
```

File: README.md [hunk 2]
```diff
@@ -17,6 +17,12 @@ To download the tool you have to have Node.js installed on your machine. After y
...
+
+```sh
+$ npm install -g speedo@latest
+```
+
```
Comment (adambabik) [inline README.md:25]: I would add a section to verify that the newest version is used with `speedo --version`.

Or even recommend:
```
$ speedo --version
// => 1.2.2
$ npm install -g speedo@latest
$ speedo --version
// => 1.2.3
```

Comment (maciejurbanskisaucelabs) [reply README.md:31]: added

File: README.md [hunk 3]
```diff
@@ -17,6 +17,18 @@ To download the tool you have to have Node.js installed on your machine. After y
 $ npm install -g speedo
 ```
 
+## Update
+
+```sh
+$ speedo --version
+// => 1.2.2
+
+$ npm install -g speedo@latest
```
Comment (christian-bromann) [inline README.md:26]: What happens if someone uses a local Speedo installation?

Comment (adambabik) [reply README.md:26]: I think then they will know what to do :) We have an installation step above with `-g` so we keep being consistent here.

**Pull request merged**